src/foam/u2/view/DayOfWeekView.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'DayOfWeekView',
+  extends: 'foam.u2.view.MultiChoiceView',
+
+  properties: [
+    {
+      class: 'foam.u2.ViewSpec',
+      name: 'choiceView',
+      value: { class: 'foam.u2.view.DayChoiceView' }
+    },
+    {
+      name: 'maxSelected',
+      value: 5
+    },
+    {
+      name: 'showMinMaxHelper',
+      value: false
+    },
+    {
+      name: 'numberColumns',
+      value: 0
+    }
+  ],
+  methods: [
+    function init() {
+      this.choices = ['Mon', 'Tue', 'Wed', 'Thur', 'Fri'];
```

Comment:
You should call this.SUPER() in init() methods in case an ancestor class is also wants to do some initialization. But, a safer approach is to define the value in your properties by adding this row:
[ 'choices', ['Mon', 'Tue', 'Wed', 'Thur', 'Fri']],
which is just short-form for:
{
  name: 'choices:',
  value: ['Mon', 'Tue', 'Wed', 'Thur', 'Fri']
}.

Reply:
Setting the value on the prop doesn't call the adapt which is what we need to go from a string array to` [[str,str],[str,str]]` format that choices needs to be in